cl/assets/templates/includes/pagination.html

```diff
@@ -1,61 +1,73 @@
 {% load humanize %}
-
-{% if results.has_previous or results.has_next %}
-<div class="well v-offset-above-3 hidden-print">
-  <div class="row">
-    <div class="col-xs-2 col-sm-3">
-      {% if results.has_previous %}
-        <div class="text-left">
-          {% if webhooks %}
-            <button id="webhooks-paginator"
-               hx-get="{% url 'webhook_events-list' format="html" %}?debug={{debug}}&page={{ results.previous_page_number }}&webhook__event_type={{ type_filter }}&event_status={{ status_filter }}"
-               hx-target="#webhook_table_body"
+{% load extras %}
+<div class="col-xs-12" >
```

Comment:
Is there a reason you wrapped this in a col-xs-12? It wasn't nested this way before, except on the docket and it usually causes weirdness unless you have col's inside row's properly.

Reply:
You marked this resolved, but it's still here?

Reply:
The "col-xs-12" is still there because it is what was there to begin with.  When the new pagination.html was created, I pulled the formatting from the old version.  The old version had a `class="col-xs-12 col-sm-3"`  In testing the responsive layouts, the col-sm-3 was deleted, leaving just the col-xs-12.

In particular, it adds whitespace above the pagination header.

I marked it as resolved because the resolution is to keep it there.

Reply:
Where do you see the col-xs-12 in the old one? The version in `main` begins:

```
{% load humanize %}

{% if results.has_previous or results.has_next %}
<div class="well v-offset-above-3 hidden-print">
  <div class="row">
    <div class="col-xs-2 col-sm-3">
      {% if results.has_previous %}
        <div class="text-left">
```

Reply:
```
{% load humanize %}

{% if results.has_previous or results.has_next %}
<div class="well v-offset-above-3 hidden-print">
  <div class="row">
    *** <div class="col-xs-2 col-sm-3"> ***
      {% if results.has_previous %}
        <div class="text-left">
```
During testing, the "col-sm-3" was found to be redundant and was removed.

Reply:
Wait, re looking.

Reply:
```
git diff 1368-update-pagination-links -- docket.html
diff --git a/cl/opinion_page/templates/docket.html b/cl/opinion_page/templates/docket.html
index a0e460a9b..012a42dfe 100644
--- a/cl/opinion_page/templates/docket.html
+++ b/cl/opinion_page/templates/docket.html
@@ -56,5 +56,9 @@
 {% endif %}
 
 
-{% include "includes/pagination.html" with page_obj=docket_entries %}
+{% if docket_entries.paginator.num_pages > 1 %}
+    <div class="col-xs-12">
+    {% include "includes/pagination.html" with results=docket_entries %}
+    </div>
+{% endif %}
 {% endblock %}
```